### ISSUE TITLE
Phantom Power Controls

### ISSUE THREAD

**Issue by kj4lxc** [wrodie/ha_behringer_mixer#77]
### Checklist

- [X] I have filled out the template to the best of my ability.
- [X] This only contains 1 feature request (if you have multiple feature requests, open one feature request for each feature request).
- [X] This issue is not a duplicate feature request of [previous feature requests](https://github.com/wrodie/ha_behringer_mixer/issues?q=is%3Aissue+label%3A%22Feature+Request%22+).

### Is your feature request related to a problem? Please describe.

Using the integration with the MR12, which is the X12 in a trench coat lol.  Everything is working beautifully but there is one thing that is missing for the system.  Each of the first 4 channels has individually controllable phantom power.  I currently have cludged together an integration for their control but I would live to not have to have 2 integrations for the same machine, and looking at your source, yours is far better in the implementation categories that mine.

### Describe the solution you'd like

The phantom power states can be changed with the OSC commands ("/headamp/01/phantom", 1) for on and ("/headamp/01/phantom", 0) for off respectively.  Ideally I would just like it if channels 1-4 would have the ability for that to be toggled so I can rid myself of my mess of an implementation.

### Describe alternatives you've considered

As I said above, I have my own implementation [https://github.com/kj4lxc/phantommixer](https://github.com/kj4lxc/phantommixer) and it just adds too much overhead and if I can help someone else expand their integration then why not?

### Additional context

No further info that I can think of, but always up for answering any questions!

**Comment by kj4lxc** [comment]
Oh I also forgot to mention, I would be willing to test any pre-builds that you have!

**Comment by wrodie** [comment]
I can look at it.
Just a couple of points/questions.
The X32 and I think the Xr12 don't have phantom power connected to a channel, it is instead associated with an input instead.  So anything I do would have to take that into consideration, eg headamp would be separated from the channel. I'm not sure that is a great issue, but wanted to mention it.

Are you able to share with me a little bit more about your use case?  I'm trying to understand why you would need phantom power control, but not something else like gain as well?

**Comment by kj4lxc** [comment]
Yes the phantom power is on the input level, each of the 4 xlr connectors on the 12 have controllable phantom power.  Having the phantom controls being separate is not a problem at all as that is the current way the our Home assistant board is set up.  As far as my use case, we have a streaming setup that uses Home assistant as its control system.  We have mics on the 1-4 connections but they are not used all the time.  When a mic is not in use we cut the phantom power to the mic as to not keep the element energized all the time, this tends to help the mic "behave" better.

**Comment by wrodie** [comment]
A new release should now be available with support for headamps.  It gives control of both gain and phantom power.
Let me know whether it works for you.

**Comment by kj4lxc** [comment]
Works great, just wanted to let you know when you built release it didnt build the actual zip for the release, only the source zips so HACS cannot install it because it is trying to download https://github.com/wrodie/ha_behringer_mixer/releases/download/v0.1.12/ha_behringer_mixer.zip  Which doesnt exist.

**Comment by wrodie** [comment]
Ah, I should check the emails more closely.  I've pushed a new release v0.1.13 that should fix those build issues (that is the only change).
